faf-commons-api/src/main/java/com/faforever/commons/api/dto/LeagueSeasonDivisionSubdivision.java

```diff
@@ -6,18 +6,23 @@
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
 
+import java.net.URL;
+
 @Data
 @ToString(onlyExplicitlyIncluded = true, callSuper = true)
 @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
 @Type("leagueSeasonDivisionSubdivision")
 public class LeagueSeasonDivisionSubdivision extends AbstractEntity<LeagueSeasonDivisionSubdivision> {
+  @ToString.Include
+  private String nameKey;
   private String descriptionKey;
   private Integer highestScore;
   private Integer maxRating;
   private Integer minRating;
-  @ToString.Include
-  private String nameKey;
   private Integer subdivisionIndex;
+  private URL imageUrl;
+  private URL mediumImageUrl;
+  private URL smallImageUrl;
```

Comment:
Please use String. URL is not suitable for data transfer objects with it's retarted exception handling.